packages/blockchain-wallet-v4-frontend/src/modals/EntitiesMigration/index.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useCallback } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { connect } from 'react-redux'
+import { bindActionCreators, compose } from 'redux'
+import styled from 'styled-components'
+
+import { Button, Icon, Modal, ModalBody, Text } from 'blockchain-info-components'
+import { actions } from 'data'
+import { ModalName } from 'data/modals/types'
+import modalEnhancer from 'providers/ModalEnhancer'
+
+const Group = styled.div`
+  margin-bottom: 20px;
+`
+const GroupHeader = styled(Text)`
+  font-size: 20px;
+  font-weight: 600;
+  color: ${(props) => props.theme.black};
+  text-align: center;
+  margin-bottom: 20px;
+`
+const GroupContent = styled(Text)`
+  font-size: 14px;
+  font-weight: 500;
+  color: ${(props) => props.theme.black};
+  text-align: center;
+`
+
+const Header = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex: 1;
+  padding: 16px;
+  justify-content: flex-end;
+`
+
+const CloseIconContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 32px;
+  height: 32px;
+  border-radius: 50%;
+  background: ${(props) => props.theme.grey000};
+  backdrop-filter: blur(54.3656px);
+  > span {
+    justify-content: center;
+  }
+`
+
+function EntitiesMigration({ cacheActions, close }) {
+  const handleCloseClick = useCallback(() => {
+    cacheActions.announcementDismissed('entities-migration')
+    close()
+  }, [cacheActions, close])
+
+  return (
+    <Modal>
+      <Header>
```

Comment:
you could probably use the `flyoutHeader` here which will probably resize correctly and then you wouldn't have to create the icon and extra styled components.